views/add-new-word.html

```diff
@@ -70,6 +70,38 @@ <h1 class="govuk-heading-xl">Add a restricted word</h1>
             name: "createdReason",
             value: createdReason
           }) }}
+          {{ govukCheckboxes({
+            name: "categories",
+            fieldset: {
+              legend: {
+                text: "Which types of waste do you transport?",
```

Comment:
Not sure we're dealing in waste! :D
Think needs to be something like `Which category or catergories are associated with this word?`

Reply:
😂 😂  I was double checking the checkbox code on govuk and clearly missed a line I need to edit. Let me get that now